bin/dotd

```diff
@@ -68,13 +68,19 @@ def open_in_default_browser(url)
   end
 end
 
+def run_on(server_name, command)
+  # escape any double quotes in the command string
+  command = %Q["#{command}"]
+  system "ssh -t gateway.code.org ssh -t #{server_name} \"#{command}\""
```

Comment:
Does ruby implicitly handle escaping pre-escaped quotes when it interpolates here?  If not, I worry you'll get

```ruby
def run_on(server_name, command) # command='my "command"'
  command = %Q["#{command}"] # command='my \"command\"'
  system "ssh -t gateway.code.org ssh -t #{server_name} \"#{command}\""
  # 'ssh -t gateway.code.org ssh -t server \"my \"command\"\"'
```